modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/jobs/VersionMigrationsJob.kt

```diff
@@ -147,9 +144,6 @@ class VersionMigrationsJob : JobService() {
         if (previousVersionCode < 1257) {
             performV7Migration()
         }
-        if (!settings.getUsedAccountManager()) {
-            performAccountManagerMigration(application)
-        }
```

Comment:
As it was over six months ago, it would be simpler not to update the account migration code below, so I removed it instead. I'm happy to add it back if you think that's a better way.

Reply:
Nope, I'm fine with that.